src/main/java/eu/europeana/metis/sandbox/repository/RecordRepository.java

```diff
@@ -47,4 +58,22 @@ public interface RecordRepository extends JpaRepository<RecordEntity, Long> {
       + "FROM RecordEntity re "
       + "GROUP BY re.datasetId")
   List<DatasetStatistic> getMetricDatasetStatistics();
+
+  /**
+   * Find provider ids by dataset id and content tier value
+   *
+   * @param datasetId The id of the dataset the records belong to
+   * @param contentTier The value of the content tier to look for
+   */
+  @Query("select re.europeanaId from RecordEntity re where re.contentTier = ?2 and re.datasetId = ?1 order by re.europeanaId asc")
```

Comment:
Something we noticed with Simon, to improve readability in SQL, is to use the keywords in Uppercase.
please update it with 
```SELECT re.europeanaId FROM RecordEntity re WHERE re.datasetId = ?1 AND re.contentTier = ?2 ORDER BY re.europeanaId ASC```
also the order of fields matter here in the sense when parsing the query, it will use the new index you specified on ```record``` table

Reply:
Fixed it